### PR TITLE
feat: add adapter execution functions

### PR DESCRIPTION
**Pull request by wu-clan**
WIP.

### CODE REVIEW

File: casbin_async_redis_adapter/adapter.py [hunk 1]
```diff
@@ -98,6 +98,10 @@ async def _delete_policy_lines(self, ptype, rule):
         else:
             await self.client.lrem(self.key, 0, json.dumps(line.dict()))
 
+    async def _update_policy_line(self, ptype, old_rule, new_rule):
+        # TODO
```
Comment (wu-clan) [inline casbin_async_redis_adapter/adapter.py:104]: Hi, @BustDot 

Can you help with this? 

I'm trying to migrate async-sqlalchemy-adapter to async-redis-adapter, but I'm missing some execution functions. I've added some, but this function is beyond my reach.

**Comment by hsluoyz** [comment]
@wu-clan plz fix conflict:

image

File: casbin_async_redis_adapter/adapter.py (hunk 1 above)
Comment (hsluoyz) [reply casbin_async_redis_adapter/adapter.py:104]: @wu-clan he may be very busy to help on this. Can you do it with the help of ChatGPT? Or @leeqvip can you help on this?

Comment (wu-clan) [reply casbin_async_redis_adapter/adapter.py:104]: I will try as much as I can.

**Comment by wu-clan** [comment]
I merged into the main branch, but one check failed because of the merge submission information.

**Comment by hsluoyz** [comment]
Replaced by: https://github.com/pycasbin/async-redis-adapter/pull/6

**Pull request closed without merging**